limited_try_bomb.py

```diff
@@ -0,0 +1,12 @@
+"""
+an example of bomb limited by number of attempts
+"""
+from glitterbomb import GlitterBomb
+
+
+bomb = GlitterBomb(name="limited", prompt_limit=5)
+secret_number = 42
+while 1:
+    decision = int(bomb.prompt("Try a number, 0-99: "))
```

Comment:
I wonder if students might think that `bomb.prompt("Try a number, 0-99: ")` will validate inputs to only accept values 0-99...